extensions/sql-migration/src/dialog/createSqlMigrationService/createSqlMigrationServiceDialog.ts

```diff
@@ -492,4 +546,10 @@ export class CreateSqlMigrationServiceDialog {
 			level: level
 		};
 	}
+
+	private enableForm(enable: boolean): void {
```

Comment:
`setFormEnabledState` or something - `enableForm` implies that this function is used for specifically enabling the form which isn't necessarily true.